pkg/gui/controllers/workspace_reset_controller.go

```diff
@@ -12,7 +12,7 @@ import (
 func (self *FilesController) createResetMenu() error {
 	red := style.FgRed
 
-	nukeStr := "reset --hard HEAD && git clean -fd"
+	nukeStr := "git reset --hard HEAD && git clean -fd"
```

Comment:
This was changed because all of the other descriptions started with `git`, except for this one.